terracumber_config/tf_files/SUSEManager-5.1-build-validation-NUE.tf

```
@@ -0,0 +1,1213 @@
+// Mandatory variables for terracumber
+variable "URL_PREFIX" {
+  type = string
+  default = "https://ci.suse.de/view/Manager/view/Manager-qe/job/manager-5.1-qe-build-validation-NUE"
+}
+
+// Not really used as this is for --runall parameter, and we run cucumber step by step
+variable "CUCUMBER_COMMAND" {
+  type = string
+  default = "export PRODUCT='SUSE-Manager' && run-testsuite"
+}
+
+variable "CUCUMBER_GITREPO" {
+  type = string
+  default = "https://github.com/uyuni-project/uyuni.git"
```

That should be spacewalk, right? [Like here](https://github.com/SUSE/susemanager-ci/blob/master/terracumber_config/tf_files/SUSEManager-5.0-build-validation-NUE.tf#L15)
```suggestion
  default = "https://github.com/SUSE/spacewalk.git"
```

---

But not sure if we use uyuni for now until the right 5.1 branch is ready.

---

Yes. See my comment in the first post:
>Product version is head for now and the repo that is used is Uyuni with the master branch. This needs to be edited near FCS once we have a new Manager 5.1 branch in spacewalk.